docs/pages/button.js

```diff
@@ -187,46 +169,34 @@ card(
         required: false,
         defaultValue: 0,
         description: [
-          'Remove the component from sequential keyboard navigation to improve accessibility. The component is not focusable with keyboard navigation but it can be focused with Javascript or visually by clicking with the mouse.',
-          `The default behaviour for the component is to be focusable in sequential keyboard navigation in the order defined by the document's source order.`,
-          `If component is disabled, the component is also unreachable from keyboard navigation.`,
+          'Use "-1" to remove Button from keyboard navigation. See the [Accessibility guidelines](/accessibility) to learn more.',
         ],
-        href: 'type-roles',
       },
       {
         name: 'role',
         type: `'button' | 'link'`,
         required: false,
         defaultValue: 'button',
-        description: [
-          `Select a button variant:`,
-          `- 'button': Use to render 'submit' or 'button'-type buttons. The button is rendered as a \`<button>\`.`,
-          `- 'link': Use for buttons to act like links. The button is rendered as an \`<a>\`.`,
-          `Required with link-role buttons.`,
-        ],
-        href: 'type-roles',
+        description: ['Use “link” to indicate Button that is acting as an `<a>` link.'],
+        href: 'roles',
       },
       {
         name: 'rel',
         type: `'none' | 'nofollow'`,
         required: false,
         defaultValue: 'none',
-        description: 'Optional with link-role buttons.',
-        href: 'type-roles',
+        description: 'Optional with link-role Buttons.',
```

Comment:
here we should link to variant rel & target.

I would follow what we did for IconButton. https://deploy-preview-1654--gestalt.netlify.app/iconbutton#role_linkProps

Reply:
link-role needs `link-role`

Reply:
also the color of the text for whatever reason is different!

Reply:
This also seems like it needs some additional work. Ticketing!